CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
+* [Windows containerd v1.7](https://github.com/Azure/AKS/issues/3975) will be the default container runtime for k8s v1.28+ on AKS Windows nodes. Windows Host Process (HPC) containers is GA in Windows containerd v1.7 and it has some [breaking changes](https://github.com/kubernetes/enhancements/tree/master/keps/sig-windows/1981-windows-privileged-container-support#container-mounts).
+* Starting Kubernetes 1.29, the default cgroups implementation on Azure Linux AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance, and read the [FAQ](https://learn.microsoft.com/troubleshoot/azure/azure-kubernetes/aks-increased-memory-usage-cgroup-v2) for cgroupsv2.
+* Staring with the `2024-01-01` and `2024-01-02-preview` APIs, we will begin to reject unknown fields in the request payloads.
+
+### Release notes
+
+* Preview Features
+  * [Artifact Streaming (Preview)](https://learn.microsoft.com/en-us/azure/aks/artifact-streaming) can now be enabled on node pools.
+
+* Bug Fixes 
+  * Under some conditions it was possible to upgrade to Azure CNI Overlay from Kubenet while using the Calico network policy. This scenario is now blocked.
+
+* Behavioral Change
+  * Increased coredns memory limits.
+  * Updated AKS add-on affinity to run on system-pool when Node Auto Provisioning is enabled.
+  * Resource group names, Azure Virtual Network names, and subnet names for clusters with [Azure CNI Overlay networking](https://learn.microsoft.com/azure/aks/azure-cni-overlay) or [Dynamic IP allocation](https://learn.microsoft.com/azure/aks/configure-azure-cni-dynamic-ip-allocation) can't be longer than 63 characters.
+
+
+* Component Updates
+  * Update Eraser version to [v1.2.3](https://github.com/eraser-dev/eraser/releases/tag/v1.2.3). This update has 3 CVE fixes:
+      * [CVE-2023-39325](https://github.com/advisories/GHSA-4374-p667-p6c8)
+      * [CVE-2023-3978](https://github.com/advisories/GHSA-2wrh-6pvc-2jm9)
+      * [CVE-2023-5043](https://github.com/advisories/GHSA-qppj-fm5r-hxr3)
+  * Azure Linux image has been updated to [Azure Linux - xxxx](vhd-notes/AzureLinux/xxxx.txt).
+  * AKS Ubuntu 22.04 image has been updated to [AKSUbuntu-xxxx](vhd-notes/aks-ubuntu/AKSUbuntu-2204/xxxx.txt).
+
```

Comment:
NPD version updated to fix CVE-xxxx. Note that this change is not tied to node image upgrade, this rollout will happen outside of the regular upgrade schedule. 
@shanalily  Can you add some of the details which CVE was fixed, the version of NPD and when the rollout started

Reply:
Good to resolve this? @aritraghosh @shanalily

Reply:
Can we add Shoshana's comment in the bugs section